### PR TITLE
[Master] profile

### PR DESCRIPTION
**Pull request by Andrew-Chew**
Updated profile to include core java grep app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ I am a recent graduate from the University of Toronto with a background in softw
 
 ## Skills
```
Comment (jarviscanada) [inline README.md:5]: I am a recent graduate from the University of Toronto (CS) with a background in software development and data analytics. I am highly adaptive to new environments and always looking for opportunities to enhance my capabilities. Currently, I am working as a Junior Software and Data Engineer at Jarvis, where I work with my team in an agile environment to deliver high-quality software products. 
I believe that the software industry is moving at an incredibly rapid pace, with innovations and technologies being discovered every day. 
I am excited to pursue my two biggest passions in software and data engineering to join this ever-evolving community, utilizing big data to design products to make the world a better and infinitely more interesting place. Softs kills...

You have work exp focus soft skills, teamwork, colb, communication, leader..

Comment (jarviscanada) [reply README.md:5]: https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ I am a recent graduate from the University of Toronto with a background in softw
...
 
-**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/GitHub, R/R Markdown, Microsoft Office Suite
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/GitHub/GitFlow, R/R Markdown, Microsoft Office Suite
```
Comment (jarviscanada) [inline README.md:7]: Microsoft Office Suite remove

Comment (jarviscanada) [reply README.md:7]: remove Google Workspace

File: README.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ I am a recent graduate from the University of Toronto with a background in softw
...
 
-**Competent:** Docker, Python, GitFlow, LateX, Google Workspace
+**Competent:** Docker, Maven, JDBC, Python, LateX, Google Workspace
```
Comment (jarviscanada) [inline README.md:9]: Python,??? Maven, JDBC, Latex, Cloud

File: README.md [hunk 4]
```diff
@@ -20,8 +20,8 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndrewChew
 **Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndrewChew/tree/master/core_java)]:
```
Comment (jarviscanada) [inline README.md:20]: **Linux Cluster Monitoring Agent** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AndrewChew/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring Agent allowing users to obtain server hardware and usage statistics in real-time. Provisioned a PostgreSQL instance using a Docker container to persist data and designed a bash agent to be installed on servers to collect and insert data into the database. Implemented  SQL queries to answer business questions. (Perforemed DA using SQL to busniess/reports). Collaborated with the development team using the Agile workflow.

File: README.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndrewChew
 
 ## Professional Experiences
 
-**Junior Software & Data Engineer, Jarvis Consulting Group (2021-present)**: Develop software using the SDLC process in an agile work environment, delivering high-quality products.
+**Junior Software & Data Engineer, Jarvis Consulting Group (2021-present)**: Develop software using the SDLC process in an agile work environment, delivering high-quality products. Spearheaded daily scrum meetings as team lead for a sprint.
```
Comment (jarviscanada) [inline README.md:51]: Responsibilities
Agile/Scrum
Soft skills, such as communication, collaboration, leadership, problem-solving, etc.

File: README.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndrewChew
...
 
 **Produce Clerk Part-Time, Loblaw Companies Limited (2016-present)**: Advise customers on over 400 products leading to improved customer satisfaction. Effectively handle conflict resolution in a sensitive and professional manner ensuring safety in the workplace. Train new employees for day-to-day operations. Successfully completed diversity training.
```
Comment (jarviscanada) [inline README.md:53]: **Produce Clerk Part-Time, Loblaw Companies Limited (2016-present)**: Advise customers on over 400 products leading to improved customer satisfaction. Effectively handle conflict resolution in a sensitive and professional manner, ensuring safety in the workplace. Train new employees for day-to-day operations. Successfully completed diversity training.

Comment (jarviscanada) [reply README.md:53]: **Events Coordinator, Hope United Church (2012-2016)**: Managed a team of three to organize after-school activities for ten children. Strategized learning sessions for grade school children in Mathematics and Science. Taught children basic piano skills.

**Review by jarviscanada — APPROVED** [review]

**Comment by Andrew-Chew** [comment]
Implemented changes to reflect comments.

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**